src/screens/AddVideo/AddVideo.component.tsx

```diff
@@ -3,13 +3,24 @@ import { Grid } from '@material-ui/core';
 import AddVideoFormik from './AddVideoFormik';
 
 interface OwnProps {
-  addVideo(videoUrl: string): void;
+  // addVideo(videoUrl: string): void;
+  addVideo(
+    videoUrl: string,
+    level: string | boolean,
```

Comment:
you can use literal types for both level and rating https://www.typescriptlang.org/docs/handbook/advanced-types.html#string-literal-types